.github/workflows/benchmark-publish.yml

```diff
@@ -0,0 +1,23 @@
+name: benchmarks publish
+on:
+  push:
+    branches: [ "master" ]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - name: checkout
+      uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with: { go-version: "1.20" }
+    - name: gobenchdata publish
+      run: go run go.bobheadxi.dev/gobenchdata@v1 action
+      env:
+        GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+        INPUT_PRUNE_COUNT: 30
+        INPUT_GO_TEST_FLAGS: -short -run=^$ # TODO: remove -short flag
```

Comment:
if too long, let's keep a short test for PRs and a long one once a day against master.

Reply:
done.